src/main/java/com/thevoxelbox/voxelsniper/voxelsniper/world/BukkitWorld.java

```diff
@@ -0,0 +1,122 @@
+package com.thevoxelbox.voxelsniper.voxelsniper.world;
+
+import com.thevoxelbox.voxelsniper.snipe.Undo;
+import com.thevoxelbox.voxelsniper.voxelsniper.biome.VoxelBiome;
+import com.thevoxelbox.voxelsniper.voxelsniper.block.BukkitBlock;
+import com.thevoxelbox.voxelsniper.voxelsniper.block.IBlock;
+import com.thevoxelbox.voxelsniper.voxelsniper.chunk.BukkitChunk;
+import com.thevoxelbox.voxelsniper.voxelsniper.chunk.IChunk;
+import com.thevoxelbox.voxelsniper.voxelsniper.entity.entitytype.VoxelEntityType;
+import com.thevoxelbox.voxelsniper.voxelsniper.location.BukkitLocation;
+import com.thevoxelbox.voxelsniper.voxelsniper.location.VoxelLocation;
+import com.thevoxelbox.voxelsniper.voxelsniper.treeType.VoxelTreeType;
+import com.thevoxelbox.voxelsniper.voxelsniper.vector.VoxelVector;
+
+import java.util.Iterator;
+import java.util.Locale;
+
+import org.bukkit.TreeType;
+import org.bukkit.World;
+import org.bukkit.block.Biome;
+import org.bukkit.entity.EntityType;
+import org.bukkit.util.BlockIterator;
+import org.bukkit.util.Vector;
+
+public record BukkitWorld(World world) implements IWorld {
+
+    @Override
+    public IBlock getBlock(VoxelLocation location) {
+        return new BukkitBlock(world.getBlockAt(location.getBlockX(), location.getBlockY(), location.getBlockZ()));
+    }
+
+    @Override
+    public IBlock getBlock(int x, int y, int z) {
+        return new BukkitBlock(world.getBlockAt(x, y, z));
+    }
+
+    @Override
+    public int getMinWorldHeight() {
+        return world.getMinHeight();
+    }
+
+    @Override
+    public int getMaxWorldHeight() {
+        return world.getMaxHeight();
+    }
+
+    @Override
+    public void setBlock(VoxelLocation location, IBlock block) {
+    }
```

Comment:
Need to figure out what to do with this one, it seems to be unused, and don't know what the intended use of this is?